firebase/deploy

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+main() {
+   set -euo pipefail
+
+   if [[ "$#" -gt 1 ]]; then
```

Comment:
you shouldn't need to quote `$#` - i hope shellcheck is not telling you to.

You could use "Arithmetic Evaluation" here instead which would be a little more natural:
```
if (( $# > 1 )); then
```